Sources/App/Core/PackageCollection+VersionResult.swift

```diff
@@ -55,6 +55,8 @@ extension PackageCollection.VersionResult {
         switch filter {
             case let .author(owner):
                 query.filter(Repository.self, \Repository.$owner, .custom("ilike"), owner)
+            case let .keyword(keyword):
+                query.filter(Repository.self, \Repository.$keywords, .custom("@>"), [keyword])
```

Comment:
I tested this and it's a case sensitive match of a full keyword, which seems like the correct approach to me 👍